cpu/xive.py

```diff
@@ -72,15 +72,26 @@ def test_storeEOI(self):
                           "missing %s / %s" % (self.hw, self.intr))
             if is_dir:
                 # If xive is a directory then read from store-eoi file
-                xive_path = "/sys/kernel/debug/powerpc/xive/store-eoi"
-
-            flags = genio.read_file(xive_path)
-            if is_dir:
-                # store-eoi value can be enabled or disable via kernel command
-                # line or using sysfs debug directory.
-                match = re.search("Y", flags) or re.search("N", flags)
+                # store_eoi_file = os.path.join(xive_path, "store-eoi")
+                xive_file = os.path.join(xive_path, "store-eoi")
+                is_file_exists = os.path.exists(xive_file)
+                if is_file_exists:
```

Comment:
if os.path.exists(xive_file):   will eleminate one line and one assigment

Reply:
> if os.path.exists(xive_file): will eleminate one line and one assigment

@abdhaleegit Done. Please check. Thanks!